### PR TITLE
Refine NextBlockBaseFee

### PR DESCRIPTION
**Pull request by kjhman21**
## Proposed changes

- Refine Implementation of #1537

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kjeom** [comment]
Clean code! LGTM.

**Review by jimni1222 — APPROVED** [review]

File: consensus/misc/kip71.go [hunk 1]
```diff
@@ -92,7 +96,7 @@ func NextBlockBaseFee(parentHeader *types.Header, config *params.ChainConfig) *b
 		if nextBaseFee.Cmp(upperBoundBaseFee) > 0 {
 			return upperBoundBaseFee
 		}
-		return makeEvenByDown(nextBaseFee)
+		return nextBaseFee
```
[the remark this answers is not in the document]
Comment (blukat29) [reply consensus/misc/kip71.go:99]: Oh, I see! Thanks.

**Review by blukat29 — DISMISSED** [review]

**Review by kjeom — APPROVED** [review]

**Review by jimni1222 — APPROVED** [review]

**Review by blukat29 — APPROVED** [review]

**Comment by kjhman21** [comment]
@aidan-kwon @jiseongnoh @yoomee1313 @mckim19 
I need one of you guys' approval. PTAL. :)

**Review by yoomee1313 — APPROVED** [review]

**Review by mckim19 — APPROVED** [review]
LGTM :)

File: consensus/misc/kip71.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jiseongnoh) [reply consensus/misc/kip71.go:99]: func NextMagmaBlockBaseFee()  returns an even number through makeEvenByFloor().

**Review by jiseongnoh — APPROVED** [review]

**Pull request merged**